tests/Earthfile

```diff
@@ -798,6 +806,10 @@ command:
     RUN echo "hello command" >./message.txt
     DO +RUN_EARTHLY --earthfile=command.earth
 
+command-explicit-global:
+    RUN echo "hello command-explicit-global" >./message.txt
```

Comment:
is this line required?

Reply:
probably not, I think I thought this is some log, but I realize it was probably used by some test from the original target (`command`).

Reply:
yeah, it's used in the `+command` test: https://github.com/earthly/earthly/blob/main/tests/command.earth#L26 
this line can be removed.